tasks/auction/simulateAuction.ts

```diff
@@ -0,0 +1,117 @@
+import { task } from "hardhat/config";
```

Comment:
added a task that does the full cycle of the auction: deployment+erc, bid, finalize. (necessary for debugging why contract didn't send the tokens to the auctioneer at the end).